### DISCUSSION TITLE
Link to pages within the "default" space

### DISCUSSION THREAD

**Discussion by biccio** [GitbookIO/community#277] [Feature Requests]
### What's your feature/idea?
Inside the Gitbook collections I can define a default space, so that I can have a "default space" URL, regardless of which one I define as default.

It would be really helpful to have the possibility to create a link to a page within the default space via the provided link widget. This way, if I change the default space, the link will not break.

This is particularly helpful when you use spaces as different versions of the same content. 

### What's your desired outcome?
Find the "default" space pages in the provided widget link. The resulting link will not have the space name inside the URL.
What's the impact of this for you?

- [ ] Critical
- [x] High
- [ ] Medium
- [ ] Low

**Comment by KristianZ** [comment]
Love this idea!

In our case, half our pages are the same across spaces. The other half is unique to each space. Currently, we're using a git automation to copy changes from shared pages to all spaces. Ideally, we could just link to pages in the default space.
